frontend/src/component/feature/FeatureView/FeatureMetrics/FeatureMetricsChart/createChartOptions.tsx

```diff
@@ -4,6 +4,17 @@ import { ChartOptions, defaults } from 'chart.js';
 import { IFeatureMetricsRaw } from 'interfaces/featureToggle';
 import { formatDateHM } from 'utils/formatDate';
 import { Theme } from '@mui/material/styles/createTheme';
+import { IPoint } from './createChartData';
+
+const formatVariantEntry = (
```

Comment:
Non trivial logic that could benefit from tests either direct unit or as part of tests for chart display or chart data preparation